EventFilter/CSCRawToDigi/plugins/CSCViewDigi.cc

```diff
@@ -250,6 +281,65 @@ void CSCViewDigi::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetu
     }
   }
 
+  if (showerDigiDump) {
+    std::cout << std::endl;
```

Comment:
Moving to `MessageLogger` would be better, although is a `edm::one::EDAnalyzer`

Reply:
Was following legacy code. Converted from cout to edm::LogVerbatim